minimesos/src/main/java/com/containersol/minimesos/cluster/MesosCluster.java

```diff
@@ -67,6 +70,8 @@ public MesosCluster(ClusterConfig clusterConfig, List<ClusterProcess> processes)
      * Recreate a MesosCluster object based on an existing cluster ID.
      *
      * @param clusterId the cluster ID of the cluster that is already running
+     * @param factory TODO(mwl)
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS1135)